README.md

```diff
@@ -10,27 +10,51 @@ One of the main objectives of this project is to supply a good starting point fo
 We're using "clean architecture" to structure, decouple, expand, and maintain the code.
 
 ## Architecture
-The architecture is based on the **Model-View-ViewModel (MVVM)** Pattern where there are 4 layers within the application:
+There are 4 layers within the application:
 - Domain layer - contains high-level abstraction of the application domain (like repositories, data access) and the use cases, which contain all of the application's business logic & domain rules.
 - Data layer - implements domain layer abstractions, the `DataSources`, related to data persistence, REST calls, etc. 
 - Device layer - implements domain layer abstractions that are not related to data persistence or user interface but are specific to the android platform: android services, cloud messaging, and many others.
-- Presentation (UI) layer - all the functionality related to the Android user interface: activities, fragments, views
+- Presentation (UI) layer - all the functionality related to the Android user interface: Composables.
 
 <p align="center">
   <img height="250" src="/images/arch.png" >
 </p>
 
-The core libraries for the communication between layer components are: [**LiveData**](https://developer.android.com/topic/libraries/architecture/livedata), used to notify the UI changes and [**Coroutines**](https://kotlinlang.org/docs/reference/coroutines-overview.html), used to perform all background tasks.
+## Overview
+In order to understand how all this work together, it's important to talk about each component role inside presentation layer.
+
+First of all, the architecture used is MVI, but in many ways it's a merging of MVI and Redux.
```

Comment:
maybe a link talking about MVI, some blog or an interesting place to take a look at it